server/routers/authenticate.ts

```diff
@@ -106,4 +181,100 @@ export const authenticateRouter = {
       },
     };
   }),
+
+  refreshPasskeySession: publicProcedure
+    .input(
+      z.object({
+        refreshToken: z.string().optional(),
+        userId: z.string().optional(),
+        deviceNonce: z.string().optional(),
```

Comment:
Is there any reason to add this as a input field rather than getting it from the header?

Reply:
actually, route was not needed.  It was left behind, bit of a duplicate from refreshSession